lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -48,13 +48,14 @@
     type: bool
     default: 'no'
     version_added: "2.8"
-  ca_cert:
+  feed_ca_cert:
     description:
       - CA certificate string used to validate the feed source SSL certificate.
         This can be the file content or the path to the file.
     type: str
-    aliases: [ importer_ssl_ca_cert ]
-  client_cert:
+    aliases: [ importer_ssl_ca_cert, ca_cert ]
+  feed_client_cert:
+    version_added: "2.10"
     description:
```

Comment:
You should mention that the default value is taken from `client_cert` if that's specified, and that this will change in Ansible 2.14.

Reply:
Thanks that's  a good idea. Added a commit for this.